### DISCUSSION TITLE
What is the purpose of "A" palettes in Radix Colors?

### DISCUSSION THREAD

**Discussion by averybuehler** [radix-ui/website#473] [General]
For example, what is the difference between `tomato` and `tomatoA`? I can see that the A palette is using `hsla` and thus has an opacity to it, but I'm still confused. It's not documented what the `A` means or when the `A` palette should be used (as compared to the regular palette). What confuses me more is that all of the `A1` colors have the same value: `hsla(0, 0%, 0%, 0)` (black with 0% opacity). On the website, however, it appears that each of them is used _on top of_ another color.

 Can someone please clarify:
1. What the purpose of the `A` color palettes are?
2. When should we be using the `A` color palette based on [the documentation](https://www.radix-ui.com/docs/colors/palette-composition/understanding-the-scale)?
3. Why are all `A1` values identical?

**Comment by benoitgrelard** [comment]
The A variants are alpha variants. This means that essentially they are to be used on top of background that may vary in colours.
They have been created in a way that makes them remain as close as possible to the original color but with baked-in alpha.
